cmd/otelcol/Dockerfile

```diff
@@ -1,16 +1,20 @@
-FROM alpine:3.12 as certs
+FROM alpine:latest as certs
```

Comment:
Looks like dependabot has support for Docker. Perhaps configure it instead of using latest to keep things reproducible?

Reply:
I just tried changing the `gendependabot` target here, but the newlines were printed as `\n` here. I changed this version then to 3.13, and a future PR can enable dependabot for this Dockerfile.